Automattic/wp-calypso
# Add rest of 2 year jetpack plan constants

## Proposed Changes

* Add new 2 year plan constants wherever else they are needed
* Most of the places added in this PR are in tests

## Testing Instructions

1. Make sure all tests pass
2. In your 0-sandbox.php file, add the following line
```php
define( 'USE_STORE_SANDBOX', true );
```
3. Sandbox `public-api.wordpress.com`
4. Using the Calypso live link or your local environment with this branch checked out, make sure you can still purchase the 2 year plans with no issues

## Pre-merge Checklist
